app/repo/userRepo.go

```diff
@@ -121,11 +121,11 @@ func (repo *userRepository) TransferRole(roleType enums.UserRole,
 
 	err := repo.coll.Find(orFilter).Select(cols).All(&users)
 	if err != nil {
-		return "", err
+		return "", nil, err
 	}
 	count := len(users)
 	if count < 2 {
-		return common.MSG_INSUFFICIENT_USER_COUNT + strconv.Itoa(count), nil
+		return common.MSG_INSUFFICIENT_USER_COUNT + strconv.Itoa(count), nil, err
```

Comment:
return common.MSG_INSUFFICIENT_USER_COUNT + strconv.Itoa(count), nil, err shoaled be changed to 
return common.MSG_INSUFFICIENT_USER_COUNT + strconv.Itoa(count), nil, nil